app/elements/challenge/challenge-modal.html

```diff
@@ -3,8 +3,14 @@
 <link rel="import" href="/bower_components/paper-input/paper-input.html">
 <link rel="import" href="/bower_components/paper-button/paper-button.html">
 <link rel="import" href="/bower_components/paper-dialog/paper-dialog.html">
+<link rel="import" href="/bower_components/paper-toast/paper-toast.html">
 
 <dom-module id="challenge-modal">
+	<style>
+		paper-toast {
+			z-index: 999999;
```

Comment:
Unfortunately it is. The thing is the dialog for challenge creation makes the page a little darker and the toast renders behind that darkness to say so. If you have any other idea regarding this aspect I will change this version (which I find a little unnatural, but it's the best I found yet).